templates/kubernetes-cluster.template

```diff
@@ -307,34 +307,36 @@ Mappings:
 
   # http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
   RegionMap:
-    us-west-1:
-      '64': ami-39023959
-    ap-south-1:
-      '64': ami-5399d73c
-    eu-west-2:
-      '64': ami-a3a1bec7
-    eu-west-1:
-      '64': ami-31d36048
-    ap-northeast-2:
-      '64': ami-0d379063
-    ap-northeast-1:
-      '64': ami-24de6742
-    sa-east-1:
-      '64': ami-bb4206d7
-    ca-central-1:
-      '64': ami-ab5ae1cf
-    ap-southeast-1:
-      '64': ami-5f0c5f3c
-    ap-southeast-2:
-      '64': ami-3882775a
-    eu-central-1:
-      '64': ami-57ec6038
-    us-east-1:
-      '64': ami-6369fa19
-    us-east-2:
-      '64': ami-e89eb08d
-    us-west-2:
-      '64': ami-7ed30d06
+      us-east-1:
+        '64': ami-957920ef
+      ap-south-1:
+        '64': ami-ea306785
+      eu-west-3:
```

Comment:
ffff

You will have to leave this one out for now until this gets closed or our CI changes: https://github.com/boto/boto/pull/3784

Reply:
Will be solved once #126 is merged